daphne_worker/src/config.rs

```diff
@@ -924,6 +924,12 @@ impl<'srv> DaphneWorker<'srv> {
                 let len = length.parse().map_err(int_err)?;
                 VdafConfig::Prio3(Prio3Config::SumVec { bits, len })
             }
+            // NOTE: the VDAF-06 format is not supported yet
+            // https://datatracker.ietf.org/doc/html/draft-dcook-ppm-dap-interop-test-design-04#name-internal-test-add_task
```

Comment:
Let's do David a favor and file an issue: https://github.com/divergentdave/draft-dcook-ppm-dap-interop-test-design/issues